Automattic/wp-calypso
# Site migration: vertical centre progress view on larger viewports

Note: This change is behind a feature flag and is part of an in-progress larger project. For context, see pbkcP4-8-p2.

#### Changes proposed in this Pull Request

* Vertically centres the migration progress view on viewports 660px and up.

![image](https://user-images.githubusercontent.com/1647564/70436051-5278b580-1a80-11ea-889f-8210adc73a9e.png)

* I've only done the vertical centring on larger viewports, as on smaller ones there's more of a danger the content will bump into the Masterbar if it ever grows:

![image](https://user-images.githubusercontent.com/1647564/70436941-5b6a8680-1a82-11ea-8c4c-c3f6b21fcb17.png)

#### Testing instructions



* Check out this pull request and run Calypso locally, or view it on calypso.live with the flag tools/migrate.
* Viewing calypso.localhost:3000/migrate/, you should see a SiteSelector component allowing you to choose a site to migrate a Jetpack site into.
* When you select a site, you should see another SiteSelector component, listing all the Jetpack sites your account has access to.
* Selecting one of those sites should take you to a confirmation screen.
* Accept the confirmation. You should see a progress screen detailing the steps of the migration. The content should be vertically centred on viewports larger than 660px.
* After the migration ends, you should be able to scroll down Calypso pages as normal. (It helps if you switch to a mobile view in your browser's dev tools to check this.)
